frontend/src/components/Table/Tab.tsx

```diff
@@ -94,17 +132,46 @@ const Tab: React.FC<TabProps> = ({ resourceLabel }) => {
                 type="primary"
                 icon={<PlusOutlined />}
                 onClick={handleAdd}
-                style={{ marginBottom: 16 }}
+                size="large"
+                style={{
+                    position: "fixed",
+                    bottom: "16px",
+                    right: "16px",
+                    zIndex: 1000,
+                    borderRadius: "50px",
+                    padding: "0 16px",
+                }}
             >
                 Add
             </Button>
+
             <Table
-                columns={columns}
+                columns={columns.map((col, index) =>
+                    index === columns.length - 1 ? { ...col, fixed: 'right' } : col
+                )}
                 dataSource={dataSource}
-                scroll={{ x: 'max-content', y: 55 * 5 }}
+                scroll={{ x: 'max-content' }}
+                pagination={{
+                    showSizeChanger: true,
+                    pageSizeOptions: ['10', '20', '50'],
+                }}
+                style={{
+                    marginTop: '64px',
+                    maxHeight: 'calc(100vh - 80px)',
+                    overflowY: 'auto',
+                }}
             />
+            {selectedRecord && (
+                <DeleteConfirmModal
+                    visible={isModalVisible}
+                    resourceName={selectedRecord.name as string}
+                    namespace={selectedRecord.namespace as string}
+                    onConfirm={handleDeleteConfirm}
+                    onCancel={handleCancel}
+                />
+            )}
```

Comment:
This doesn’t work for me. It might be a good idea to move the modal higher, into `ResourcePage`. That way, you’d only need to pass showModal through props. I did something similar in my Helm branch—you can check how that works if you want. I’m specifically referring to rollback; delete still needs to be reworked in that branch.

Reply:
I thought about this. I didn't move the confirmation modal to a separate page because I consider it not an independent page but just a component that takes data and confirms actions. After the PR is merged, we can compare both of our solutions.